thirdparty/assimp/include/assimp/vector3.h

```diff
@@ -124,6 +124,14 @@ class aiVector3t {
      *  Note that vec*vec yields the dot product.
      *  @param o Second factor */
     const aiVector3t SymMul(const aiVector3t& o);
+    
+    static aiVector3t<TReal> NORMAL() {
+        return aiVector3t<TReal>(1,1,1);
```

Comment:
Constants tend to be using 1.0f syntax to avoid the conversion of double to float. However TReal is a type def.. and this is assimp code.

Reply:
yeah in assimp that can be a double, float or whatever you want.

I left it at 1 purely because this is what they do in other parts of assimp.

The code modifications to assimp and extensions are on purpose, the pivot implementation is pretty much only going to be in assimp but I am working on it using godot, because this is easier than working in assimp's model viewer